payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherViewModel.kt

```diff
@@ -125,7 +145,10 @@ internal class PaymentLauncherViewModel @Inject constructor(
                     }
                     if (!intent.requiresAction()) {
                         withContext(uiContext) {
-                            internalPaymentResult.value = InternalPaymentResult.Completed(intent)
```

Comment:
We wanted a common place to handle analytics. So I moved that logic to `postInternalResult`, which is the only place `internalPaymentResult` is updated.